### PR TITLE
Minor address page tweaks

### PR DESCRIPTION
**Pull request by GBKS**
Reordered address formats to place newest ones at the top.
Added a link to an address validation tool where readers can try out validation of SegWit and Bech32 addresses.

🔆[Peep the preview](https://deploy-preview-830--sad-borg-390916.netlify.app/guide/glossary/address/)🔅

### CODE REVIEW

File: guide/glossary/address.md [hunk 1]
```diff
@@ -45,21 +45,15 @@ When users enter an address, these formats have specific prefixes, so it is poss
...
 
-### Script address - [P2SH](https://en.bitcoin.it/wiki/Pay_to_script_hash)
+A pay-to-taproot (P2TR), also known as a Taproot or Bech32m address, is the most recent and advanced bitcoin address format. Taproot introduces more advanced security, privacy, flexibility and scaling to bitcoin. Like SegWit, Taproot addresses are opt-in and not currently widely supported. Taproot adoption can be tracked [here](https://en.bitcoin.it/wiki/Bech32_adoption).
 
-A pay-to-script-hash (P2SH), or script address, can have additional rules and functionality attached to the address. Script addresses are commonly used for multi-sig addresses, which can specify that signatures from several keys are required to authorize the transaction.
+Benefits of Taproot include the ability to use Schnorr Signatures offering better security, lower fees, and more flexible [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) transactions. Multi-key addresses using P2TR look the same as single-key addresses giving users of multi-key increased privacy. Taproot also enables more advanced scripting, enabling more complex smart contracts to be built on bitcoin.
```
Comment (sbddesign) [inline guide/glossary/address.md:52]:
```suggestion
Benefits of Taproot include the ability to use Schnorr Signatures, offering better security, lower fees, and more flexible [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) transactions. Multi-key addresses using P2TR look the same as single-key addresses giving users of multi-key increased privacy. Taproot also enables more advanced scripting, enabling more complex smart contracts to be built on bitcoin.
```

File: guide/glossary/address.md [hunk 2]
```diff
@@ -71,15 +65,21 @@ SegWit addresses start with `bc1q` and are case insensitive.
...
-Benefits of Taproot include the ability to use Schnorr Signatures offering better security, lower fees, and more flexible [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) transactions. Multi-key addresses using P2TR look the same as single-key addresses giving users of multi-key increased privacy. Taproot also enables more advanced scripting, enabling more complex smart contracts to be built on bitcoin.
+Script addresses start with the number 3, can contain upper and lowercase characters, and are case sensitive.
 
-Taproot addresses start with `bc1p` and are case insensitive.
+Example: `3J98t1WpEZ73CNmQviecrnyiWrnqRhWNLy`
 
-Example: `bc1pmzfrwwndsqmk5yh69yjr5lfgfg4ev8c0tsc06e`
+### Legacy address - [P2PKH](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash)
+
+A pay-to-pubkey-hash (P2PKH), or legacy address, is the oldest and original bitcoin address format. This address format is not widely used today as transaction costs are higher using this format. However, support for this in applications is still encouraged as to be compatible with older legacy wallets that have not upgraded to new address formats.
```
Comment (sbddesign) [inline guide/glossary/address.md:78]:
```suggestion
A pay-to-pubkey-hash (P2PKH), or legacy address, is the oldest and original bitcoin address format. This address format is not widely used today as transaction costs are higher using this format. However, support for this in applications is still encouraged to be compatible with older legacy wallets that have not upgraded to new address formats.
```

**Review by sbddesign — DISMISSED** [review]
Looks good, minor grammar edits.

**Comment by GBKS** [comment]
Thanks for the review. I actually didn't edit any of that copy, had just moved it around. Still good to fix these.

**Review by Bosch-0 — APPROVED** [review]
LGTM!

**Pull request merged**